src/main/plugins/browser-histories-plugin/browser-history.ts

```diff
@@ -0,0 +1,4 @@
+export interface BrowserHistory {
```

Comment:
I'd call this interface `BrowserHistoryEntry` or something similar. As far as I understand the "history" is a collection of entries, and this interface represents the single entry, right?